### DISCUSSION TITLE
leading-snugish tailwind class?

### DISCUSSION THREAD

**Discussion by riekusr** [WebPraktikos/universal-resume#16] [General]
Hi mate, awesome project! Really appreciate your efforts, just a quick question. What is the "leading-snugish" class do? Can't seem to find any reference to it and google doesn't help either. 

Thanks!

**Comment by riekusr** [comment]
Sorry, already found it, it is depreciated use of leading-snug.

**Comment by WebPraktikos** [comment]
Hi there! It's a custom `lineHeigth` property as you can here: https://github.com/WebPraktikos/universal-resume/blob/master/tailwind.config.js
